truss-chains/examples/transcribe/transcribe.py

```diff
@@ -98,13 +99,13 @@ def __init__(
 
     async def run_remote(
         self,
-        media_url: str,
-        whisper_parmas: data_types.WhisperParams,
-        params: data_types.TranscribeParams,
+        url: str,
```

Comment:
Why shorten `media_url`? It's not an overly long name. There could be other URLs such as webhooks or model predict URLs... I don't think the extra specifier here hurts.